### PR TITLE
Accessing FIXSession inside FIXStatusListener

### PR DESCRIPTION
**Pull request by jheusser**
As discussed in #10 

Ran tests and initiators/acceptors for ping pong test.

### CODE REVIEW

File: philadelphia/src/main/java/org/jvirtanen/philadelphia/FIXStatusListener.java [hunk 1]
```diff
@@ -10,57 +10,64 @@
     /**
      * Receive an indication to close the connection.
      *
+     * @param session the Session
```
Comment (jvirtanen) [inline philadelphia/src/main/java/org/jvirtanen/philadelphia/FIXStatusListener.java]: A small nit, but for uniformity, could you use lowercase for "session" (cf. "message" below)? Thanks for bearing with me. :grin:

**Comment by jvirtanen** [comment]
Looks very good!

**Comment by jheusser** [comment]
Makes sense, updated with javadoc adjustment.

**Comment by jvirtanen** [comment]
Merged, thank you!

**Comment by jheusser** [comment]
:+1:

**Pull request merged**